apps/src/templates/instructions/teacherFeedback/EditableReviewState.jsx

```diff
@@ -66,27 +66,30 @@ class EditableReviewState extends Component {
 
   render() {
     return (
-      <div style={styles.keepWorking}>
+      <div
+        style={styles.keepWorking}
+        data-tip
```

Comment:
For my education, why did we need to move these data properties up to this div?

Reply:
I needed the element with the `data-tip` properties is the one that will show the tooltip when a user hovers on it. I needed it to wrap the checkbox as well as the keep working text, before it was just wrapping the keep working text.